test/connectors/VatSubscriptionConnectorSpec.scala

```diff
@@ -45,70 +44,31 @@ class VatSubscriptionConnectorSpec extends SpecBase with MockitoSugar with Scala
 
     "mandationStatus is called" should {
 
-      "return a valid Right(Some(MandationStatus))" in {
-        val mtdMandated = MandationStatus("MTDfB Mandated")
+      "return Right(true) when the call is successful (200)" in {
 
-        val response: Future[Either[String, Option[MandationStatus]]] =
-          vatSubscriptionConnector(mockedResponse = HttpResponse(OK, Some(Json.toJson(mtdMandated))))
+        val response: Future[Either[String, Boolean]] =
+          vatSubscriptionConnector(mockedResponse = HttpResponse(OK, Some(Json.toJson("Non MTDfB"))))
```

Comment:
Does it have to be "Non MTDfB"?
If so then make it a constant and add some validation in the connector
If all we care about is the status code then don't put anything in the body